File: src/media/MediaType.java
package media;

/**
 * Created by Timothy on 7/7/19.
 */
public enum MediaType {
    // TODO(stfinancial): Should this also be used for album artwork?
    FLAC(".flac"),
    MP3(".mp3"),
    WAV(".wav");

    private String extension;

    private MediaType(String extension) {
        this.extension = extension;
    }

    public String getExtension() { return extension; }
}

File: muzak/media/MediaType.cs
namespace muzak.media
{
    class MediaType
    {
        // TODO(stfinancial): Should this also be used for album artwork?
        public static readonly MediaType FLAC = new MediaType(".flac");
        public static readonly MediaType MP3 = new MediaType(".mp3");
        public static readonly MediaType WAV = new MediaType(".wav");

        public string Extension { get; private set; }

        private MediaType(string extension) => (Extension) = (extension);
    }
}
